queries.sql

```diff
@@ -1,12 +1,46 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * FROM animals WHERE name LIKE '%mon';
+
 SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;
+
 SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
+
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu'); 
+
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
 SELECT * FROM animals WHERE neutered = true; 
+
 SELECT * FROM animals WHERE name NOT IN ('Gabumon'); 
+
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE name NOT LIKE '%mon';
+COMMIT;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
+
+SELECT neutered, MIN(weight_kg) FROM animals GROUP BY neutered;
+
+SELECT neutered, MAX(weight_kg) FROM animals GROUP BY neutered;
 
+SELECT AVG(escape_attempts) FROM animals
+WHERE EXTRACT(year FROM date_of_birth) BETWEEN 1990 AND 2000;
```

Comment:
- [ ] I think the question here is `What is the average number of escape attempts per animal type of those born between 1990 and 2000?` so I think it would be better if you also select the species as well with the AVG(escape_attempts) so kindly update it and also you should GROUP BY species because it say `per animal type of those` so please update it ✅😊